src/detail-page/DetailPage.tsx

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
 import { useParams } from 'react-router-dom';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 import { MainContainer } from '../layout/components/MainContainer';
 import { Tabs } from './components/tabs/Tabs';
 
 import { ITProfessions } from '../data/ITProfessions';
+import { ProfessionCard } from '../shared/components/ProfessionCard';
 
 export const TabContainer = styled('div')`
-    max-width: 960px;
-    width: 80%;
-    margin: 2em auto 0;
-    padding-bottom: 2em;
+  ${({ theme }) => css`
+    width: 100%;
+    margin: ${theme.spacing(4, 0, 0)};
+    padding-bottom: ${theme.spacing(4)}px;
 
-    ${props => props.theme.breakpoints.down('sm')} {
-      width: 90%;
-      margin-top: 1.5em;
+    ${theme.breakpoints.down('sm')} {
+      margin: ${theme.spacing(3, 0, 0)};
     }
+  `}
```

Comment:
I changed spacing here to defaut theme spacing. We may later discuss with all girls if we want to use absolute or relative values in spacing and if we want to change something more - adjust theme.